src/log.js

```diff
@@ -412,16 +412,16 @@ export class Log {
    *
    * Otherwise see `assert` for usage
    *
-   * @param {*} shouldBeElement
+   * @param {?} shouldBeElement
    * @param {!Array|string=} opt_message The assertion message
    * @return {!Element} The value of shouldBeTrueish.
-   * @template T
    * @closurePrimitive {asserts.matchesReturn}
    */
   assertElement(shouldBeElement, opt_message) {
+    const shouldBeTrueish = shouldBeElement && shouldBeElement.nodeType == 1;
     this.assertType_(
       shouldBeElement,
-      isElement(shouldBeElement),
+      shouldBeTrueish,
```

Comment:
Is this revert intentional?

Reply:
Not at all! Must have gotten out of date with main with a bad rebase.
I just ran `git checkout main src/log.js` and reapplied the necessary changes.